_pages/about-us/teams/products-and-platforms.md

```diff
@@ -51,10 +53,10 @@ If you're a P&P project lead, expect to:
 
 * Report weekly on your project's status, blockers, and financial status via the weekly meeting agenda.
 
-* Be involved in the staffing process
+* Be involved in the staffing process.
```

Comment:
That sounds good. And to clarify my comment: I don't think this needs to be extremely granular, but it could benefit from including a few more specifics (e.g., identifying appropriate team members for given projects, alerting team members to new project assignments, what have you).